python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -101,6 +99,32 @@ def parse_node_data(in_file, feat_ops, label_ops, node_id_col, read_file):
     node_ids = data[node_id_col] if node_id_col in data else None
     return (node_ids, feat_data)
 
+def prepare_edge_data(in_file, feat_ops, read_file):
+    """ Parse node data.
```

Comment:
we need to change the docstring. how is this different from `parse_edge_data`?

Reply:
Updated